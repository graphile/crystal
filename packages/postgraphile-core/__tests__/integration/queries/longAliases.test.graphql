query {
  person: personByEmail(email: "sara.smith@email.com") {
    nodeId
    email

    # safe alias: @@25fa9871b4d4d16ffd41359c88e7e851739819c6
    aVeryLongAliasInOrderToTestThatThingsGoThroughPostgresJustFineAndWeDontGetAnyIssuesDueToAliasLength: friends {
      totalCount
    }

    # safe alias: @@e82261e340b3c5fc784bd0d54ec53541f5a4e2fe
    anotherLongAliasInOrderToTestThatThingsGoThroughPostgresJustFineAndWeDontGetAnyIssuesDueToAliasLength: friends {
      totalCount
    }
  }
}
