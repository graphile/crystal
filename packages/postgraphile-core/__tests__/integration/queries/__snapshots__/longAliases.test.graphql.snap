// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "person": Object {
      "aVeryLongAliasInOrderToTestThatThingsGoThroughPostgresJustFineAndWeDontGetAnyIssuesDueToAliasLength": Object {
        "totalCount": 2,
      },
      "anotherLongAliasInOrderToTestThatThingsGoThroughPostgresJustFineAndWeDontGetAnyIssuesDueToAliasLength": Object {
        "totalCount": 2,
      },
      "email": "sara.smith@email.com",
      "nodeId": "WyJwZW9wbGUiLDJd",
    },
  },
}
`;
